docs/en/ingest-management/elastic-agent/configuration/elastic-agent-monitoring.asciidoc

```diff
@@ -38,6 +38,11 @@ To enable monitoring, set `agent.monitoring.enabled` to `true`. Also set the
 collected. If neither setting is specified, monitoring is turned off. Set
 `use_output` to specify the output to which monitoring events are sent.
 
+Setting `agent.monitoring.enabled` to `true` also exposes a `/liveness` endpoint. By default, the endpoint 
```

Comment:
It's actually `agent.monitoring.enabled.http` that needs to be enabled, `agent.monitoring.enabled` controls whether we collect monitoring data from agent to be shipped to Fleet.

The liveness probe is an HTTP feature. Sorry for the late review.

Reply:
Thanks for spotting this @cmacknz!

A couple of questions:
 - Should it be `agent.monitoring.enabled.http` or `agent.monitoring.http.enabled`? In your [comment](https://github.com/elastic/elastic-agent/issues/107#issuecomment-2133910470) you mentioned the latter.
 - I don't think the setting is available to in Fleet, so I think I should mention the setting only in the standalone agent docs ([here](https://www.elastic.co/guide/en/fleet/8.13/elastic-agent-monitoring-configuration.html)). Does that seem right to you?